src/foundation.cs

```diff
@@ -8780,6 +8780,22 @@ partial interface NSItemProvider : NSCopying {
 #endif
 	}
 
+	interface INSItemProviderReading {}
+	
+	[Watch (4,0), TV (11,0), Mac (10,13), iOS (11,0)]
+	[Protocol, Model]
```

Comment:
I think this protocol is supposed to be implemented by existing types that can be dragged/dropped, in which case `[Model]` doesn't make sense, it can be only `[Protocol]`.